features/step_definitions/background_steps.rb

```diff
@@ -0,0 +1,9 @@
+Given("I am on the Login page") do
+    visit user_session_path
+  end
+  
+Given("the following user exists") do |table|
+  table.hashes.each do |hash|
+    User.create!(hash)
```

Comment:
please use factory bot here
`create(:user, hash)`
or
`FactoryBot.create(:user, hash)`

there is a way of generating a factory. 
`rails g factory_bot:model Car name:string email:string password:string`